### DISCUSSION TITLE
Can't CaptionButtons set templates?

### DISCUSSION THREAD

**Discussion by wmchuang** [AvaloniaUI/Avalonia#11023] [General]
> <Style Selector="CaptionButtons">
		<Setter Property="Template">
			<ControlTemplate>
				<Border Background="Red" Width="50" Height="50"></Border>
			</ControlTemplate>
		</Setter>
	</Style>

I am using version 0.10.19, and I want to reset the Template for CaptionButtons to replace the Background with #7FFF0000 when Panel:pointerover occurs, but it seems that this is not possible. If this is not possible, how can I modify its color? Can anyone help me?

**Comment by maxkatz6** [comment]
It is possible https://github.com/AvaloniaUI/Avalonia/blob/0.10.19/src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml
You can copy paste whole style and adjust or just add specific styles for color only.
